zephyr/modules/owntech_data_api/zephyr/public_api/DataAPI.cpp

```diff
@@ -249,6 +276,55 @@ void DataAPI::setParameters(uint8_t adc_num, uint8_t pin_num, float32_t gain, fl
 }
 
 
+int8_t DataAPI::saveParametersInNVS(uint8_t adc_num, uint8_t pin_num)
+{
+	uint8_t channel_num = this->getChannelNumber(adc_num, pin_num);
+	if (channel_num == 0)
+	{
+		return -5;
```

Comment:
What does this value mean? If this is a special value, same as above: define a constant in header.